Adyen/UI/Form/Items/Toggle/FormToggleItemView.swift

```diff
@@ -36,7 +36,7 @@ public final class FormToggleItemView: FormItemView<FormToggleItem> {
         switchControl.translatesAutoresizingMaskIntoConstraints = false
         switchControl.isOn = item.value
         switchControl.onTintColor = item.style.tintColor
-        switchControl.isAccessibilityElement = false
```

Comment:
Shouldn't the current default values stay that way?

Reply:
My thinking was that we make it testable by default and then VoiceOver notification triggers logic change. Do you see any potential hiccups here?